src/core/agent/records/credentialMetadataRecord.types.ts

```diff
@@ -10,6 +12,8 @@ interface CredentialMetadataRecordProps {
   status: CredentialStatus;
   connectionId: string;
   schema: string;
+  identifier: IdentifierMetadataRecordProps;
```

Comment:
We just want to store the identifier ID. Not the props used when creating the identifier record